gamestonk_terminal/portfolio/README.md

```diff
@@ -0,0 +1,40 @@
+#PORTFOLIO
+
+This was developed to interface with Robinhood.  
+
```

Comment:
Regarding the structure under I would divide it in sub-menus. Although currently we only support robinhood, it is likely that this will be extended in the future.

Having something like https://github.com/DidierRLopes/GamestonkTerminal/tree/main/gamestonk_terminal/behavioural_analysis
where we separate commands based on whether they belong from Twitter, Google, Stocktwits, Reddit, ... is probably nicer. #thisistheway.